### DISCUSSION TITLE
add topic on your repo

### DISCUSSION THREAD

**Discussion by Mika3578** [ansani/Shareaza#65] [Ideas]
can you add a lot of topics on your fork to have better view and get more people here 😉

like this 
<img width="517" alt="topic" src="https://user-images.githubusercontent.com/58137747/201492379-8667c708-f168-4345-bb89-f2c014ee8846.png">

more topics to suggest are welcome from user read this...

**Comment by ansani** [comment]
great idea and thank you! done! :)

**Reply by Mika3578** [reply]
more soon

**Comment by Mika3578** [comment]
DirectConnect
DC++
GnutellaG2
eDonkey

**Comment by ansani** [comment]
> DirectConnect
> 
> DC++
> 
> GnutellaG2
> 
> eDonkey
> 
> 

Cool! 🎊
